components/filter/test/filter-tags.vdiff.js

```diff
@@ -136,6 +136,19 @@ describe('filter-tags', () => {
 			await expect(elem).to.be.golden();
 		});
 
+		it('clear-one-filter-set', async() => {
+			const elem = await fixture(tagsTwoFiltersMoreSelected, { viewport: { width: 1700 } });
+
+			for (let i = 0; i < 2; i++) {
+				await sendKeysElem(elem.querySelector('d2l-filter-tags'), 'press', 'Tab');
+				await sendKeys('press', 'ArrowRight');
+				sendKeys('press', 'Delete');
+			}
```

Comment:
I don't know that this test is necessarily testing what we want. Based on looking at `tagsTwoFiltersMoreSelected` it looks like only one filter value is selected from `filter-1` so all we'd want to do is delete the first item and confirm that focus goes to the second item.